shell/platform/android/io/flutter/embedding/engine/renderer/FlutterRenderer.java

```diff
@@ -200,34 +241,31 @@ protected void finalize() throws Throwable {
           return;
         }
 
-        handler.post(new SurfaceTextureFinalizerRunnable(id, flutterJNI));
+        handler.post(
+            new Runnable() {
+              @Override
+              public void run() {
+                if (!flutterJNI.isAttached()) {
+                  return;
+                }
+                Log.v(TAG, "Releasing a SurfaceTexture (" + id + ").");
```

Comment:
Please remove this log - it's not really useful to end developers and will just end up adding to noise.

Reply:
done.